src/js/experimental/media-captions-listbox.js

```diff
@@ -135,6 +185,27 @@ class MediaCaptionsListbox extends MediaChromeListbox {
         detail: this.value,
       }
     );
+
+    const preferredLanguage = parseTextTrackStr(this.value)?.language;
+    if (preferredLanguage) {
+      const mediaController = this.hasAttribute(
+        MediaStateReceiverAttributes.MEDIA_CONTROLLER
+      )
+        ? // @ts-ignore
+          closestElementSelector(
+            this,
+            `#${this.getAttribute(
+              MediaStateReceiverAttributes.MEDIA_CONTROLLER
+            )}`
```

Comment:
I would keep the id linking contained in the same shadow DOM I think.
that's how native HTML also behaves afaik. thoughts?

Reply:
ah this is actually another thing that I think can be completely removed after my refactor to support toggle requests, but I'll confirm. If not, we are in a tricky situation with well-defined nested component cases (e.g. in this case, `media-captions-selectmenu` having a `media-captions-listbox` in its shadow DOM. We could, ofc, pass the value down instead? Either way, I think this should be irrelevant and not a thing we need to hash out right now(🤞)

Reply:
no longer relevant bc no longer needed 🎉